### DISCUSSION TITLE
Educational material in Spanish (Soroban)

### DISCUSSION THREAD

**Discussion by Naiclos** [stellar/sorobounty-spectacular#31] [General]
Hello! Hoping you are doing well, I am sharing my educational work on Soroban for the Spanish-speaking community, where you can find:

- [GitHub repository](https://github.com/Naiclos/soroban-quest) with information on all Soroban Quests translated into Spanish, Readme files, and Prologues of the story.
- [Video tutorial on YouTube ](https://www.youtube.com/watch?v=SsVM_Ma1dE8) on how to set up the development environment and the first steps to solve the Hello-World quest in Spanish, as well as a clear description of the commands and steps to follow.   

The objective of this work is to teach in Spanish that Soroban is a native smart contract platform on the Stellar network, enabling developers to create and deploy smart contracts efficiently and securely. With Soroban in Spanish, developers in Latin America can leverage Stellar's blockchain technology to build decentralized applications, protocols, and resources targeted at areas with limited access to information and financial education. This way, having content in Spanish directly contributes to the digitalization and expansion of the community and the network.

**Comment by fazzatti** [comment]
Awesome material @Naiclos ! 

My main channel [HiFifo ](https://www.youtube.com/@hififo)is in English but I also have one ([OiFifo](https://www.youtube.com/@oififo)) focused on Portuguese speakers in Latin America.

I'll definitely connect to your channel so I can also refer it to the Latam community!

**Reply by Naiclos** [reply]
Thank you for your words @fazzatti ! It's a pleasure to be able to connect through this Soroban material in Spanish. I will be checking out your channel! And I will let you know when I have more material in Spanish to contribute!

**Comment by briwylde08** [comment]
Hi @Naiclos! Love the Spanish content. Unfortunately, submissions for Sorobounty Spectacular closed on August 28th. I'm sure we'll have more Soroban Adoption Fund programs that you'll be able to participate in coming up though! 

Let me know if you have any questions!

Best,
Bri
